lib/Relationship.php

```diff
@@ -708,6 +708,8 @@ class BelongsTo extends AbstractRelationship
 {
     public function __construct($options=[])
     {
+        private $primary_key;
```

Comment:
Is this part of the fix?

Reply:
Yes. Same idea as the
private $klass
fix